demo/uhtml/index.html

```diff
@@ -2,19 +2,26 @@
 <html>
   <head>
     <meta charset="utf-8">
-    <!-- TODO: The uhtml library has a bunch of dependencies which makes the CSP
-          unwieldy. Rather than enumerate them all, a blanket rule is encoded
-          below to allow any access to https://esm.sh/. You wouldn’t want to do
-          that on a production site, this is just for the demo. -->
-    <meta http-equiv="content-security-policy" content="default-src 'self'; script-src 'self' 'sha256-tqJYP2BfE6OLuQk1TacH2iYQga16y7fS413atm5uM9o=' https://esm.sh/;">
+    <meta http-equiv="content-security-policy" content="
```

Comment:
the people must know

Reply:
Fair enough. I was definitely waffling on this one.

Reply:
For sure, I had just stopped when I was pasting `%`-encoded cruft. Will it work if we rewrite it as `'https://esm.sh/udomdiff@^1.1.0'`?

Reply:
oh yeah, noice